### ISSUE TITLE
FileNotFoundError: [WinError 2]  The system could not find the specified file

### ISSUE THREAD

**Issue by z972778371** [aimagelab/meshed-memory-transformer#69]
It seems that there are some files missed in the working environment m2release of my computer.But I downloaded the annotations.zip and the coco_detections.hdf5(53.5GB) noted in the Readme file.What did I miss?
![image](https://user-images.githubusercontent.com/51855588/138564600-f6714ded-8fd6-4983-aac0-c77b4bbe06a8.png)

**Comment by amazingYX** [comment]
What kind of operating system do you use？windows or linux?

**Comment by z972778371** [comment]
> What kind of operating system do you use？windows or linux?

windows10 
I solve this problem,but I find there's something wrong with evaluation folder's code
![138787311-d367ed17-d887-499e-a647-843b2386b2e1](https://user-images.githubusercontent.com/51855588/140597683-e524c04d-021b-4cc7-8489-c8d42ab9a9af.png)
.Have you ever met this problem before?

**Comment by amazingYX** [comment]
Yes, I meet the same problem. At first I thought it was the difference between windows10 and linux that caused this problem. Fortunately, I overcome this problem by adding absolute path in tokenizer.py .

**Comment by amazingYX** [comment]
Yes, I meet the same problem. At first I thought it was the difference between windows10 and linux that caused this problem. Fortunately, I overcome this problem by adding absolute path in tokenizer.py .

**Comment by z972778371** [comment]
> Yes, I meet the same problem. At first I thought it was the difference between windows10 and linux that caused this problem. Fortunately, I overcome this problem by adding absolute path in tokenizer.py .

Is the path_to_jar_dirname variable ？
![image](https://user-images.githubusercontent.com/51855588/140602333-35bd8a16-5fc7-436a-b9f1-40af9ac0fe74.png)

**Comment by amazingYX** [comment]
just like  this

**Comment by amazingYX** [comment]
just like this
![image](https://user-images.githubusercontent.com/44744248/140644818-bc5094c5-5227-4bd4-a040-b1e54705e5d8.png)

**Comment by amazingYX** [comment]
Have you ever reproduce DLCT's code successfully?

**Comment by z972778371** [comment]
> Have you ever reproduce DLCT's code successfully?

Not yet. I find the reason for OSError :[Error 22]it's that the command, java -jar -Xmx2G meteor-1.5.jar - - -stdio -l en -norm, can't work right in win. So I blocked the meteor() in init.py and it's finally worked.
Now I'm going to debug the DLCT's code. 囧

**Comment by amazingYX** [comment]
> > Have you ever reproduce DLCT's code successfully?
> 
> Not yet. I find the reason for OSError :[Error 22]it's that the command, java -jar -Xmx2G meteor-1.5.jar - - -stdio -l en -norm, can't work right in win. So I blocked the meteor() in init.py and it's finally worked. Now I'm going to debug the DLCT's code. 囧

When using it's Spice() to compute spice score in meshed-memory transformer , I am getting this error, subprocess.CalledProcessError: Command returned non-zero exit status 1. Have you ever met and solved this problem?

**Comment by ccccchenllll** [comment]
> 

HELLO, I have changed the code to what you said, but there are still errors. Could u explain in detail how to change it, please?
![image](https://user-images.githubusercontent.com/88440133/212636111-baa1cc78-caa0-499d-94e0-dda7591bf0d1.png)
